core/domain/exp_jobs_one_off.py

```diff
@@ -363,3 +363,60 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         yield (key, values)
+
+
+class FallbacksToHintsOneOffJob(jobs.BaseMapReduceJobManager):
+    """Job that converts existing Fallbacks to Hints for explorations with
+    state schema version 10.
+
+    IMPORTANT: The GAE memcache must be flushed after running this job.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+        if item.states_schema_version != 10:
```

Comment:
OK. I think the job probably isn't necessary actually, because the conversion function didn't go out in the last release (right?) So no explorations have been converted yet, and changing the conversion function (as you've done) should suffice. Is that correct?